src/transform-sdk/js/js_vm.cc

```diff
@@ -406,10 +451,27 @@ std::expected<std::monostate, exception> runtime::create_builtins() {
     console_builder.method<&console::warn>("error");
     auto factory = console_builder.build();
     auto global_this = value::global(_ctx.get());
-    return global_this.set_property(
+    auto result = global_this.set_property(
       "console",
       factory.create(std::make_unique<console>(
         console::config{.info = stdout, .warn = stderr})));
+
+    if (!result.has_value()) {
+        return result;
+    }
+
+    auto process_builder = class_builder<process>(_ctx.get(), "Process");
```

Comment:
Can we not create a class but just make a plain object with a single property of `env`? I don't see the value of a class here.

Reply:
1. just realized dot notation can be used interchangeably with brackets for arbitrary objects 🤦 
2. I was thinking that people might come to expect some of the other junk offered by [Process](https://nodejs.org/api/process.html), but it won't be hard to add the class later if needed, esp. given (1).

Reply:
1. It took me time to realize this too. Magic of JS!
2. [YAGNI](https://martinfowler.com/bliki/Yagni.html), but yes it's not hard to add if so